_i18n/en.yml

```diff
@@ -501,6 +501,15 @@ faq:
   avuln: >
     The Monero community has created a series of videos called "Breaking Monero", where potential Monero vulnerabilities are explored and discussed. There are 14 videos, with each exploring a different subject. Check out <a href="https://www.monerooutreach.org/breaking-monero/">the videos on monerooutreach.org</a>.
   vulnspotify: Available on Spotify as podcast
+  qcriminals: Is Monero for criminals?
+  acriminals: No. Monero is permissionless, peer-to-peer and decentralized. This means Monero developers and community have no control over how is used and who uses the technology. Monero is a tool in the hands of the public, very much like a kitchen knife or the US dollar, both of which are currently used to commit or fund horrible crimes, like terrorist attacks.
+  acriminals1: >
+    Monero, like every other currency, is money and can be used with good intentions (like funding charities) or bad intentions (like criminal activities). We cannot decide who uses Monero, but that doesn't mean we support criminal activities. Exactly like the Dollar is not considered 'for criminals', even if it's the preferred currency of terrorists, drug cartels and countless other criminal organizations.
+  qbanmonero: If you cannot avoid criminals using Monero, shouldn't it be banned?
+  abanmonero: Banning Monero would be a minor annoyance for criminals, but a major problem for those who use this currency to escape war, mass surveillance or sexual/political discrimination by tyrannical regimes or organizations.
+  abanmonero1: Banning Monero is impossible by design, but even if it was possible, the only consequence of banning it would be the sudden rise of dozens of Monero clones. Exactly how it happened with Silk Road, the pioneer darknet market. Closing it resulted in countless more darknet markets being created, many of which didn't share the ethos of the original Silk Road
+  abanmonero2: "Another example is decades of war on drugs, still ongoing in most of the world. The result of this war is in front of everyone's eyes: countless of casualties and ruined lives, but drugs have never been so many and easier to find."
```

Comment:
```suggestion
```
This one should also be removed. Monero community has no consensus about war on drugs.

Reply:
This is not something the community needs to reach consensus on. It's history.